temporalcli/commands.env_test.go

```diff
@@ -0,0 +1,40 @@
+package temporalcli_test
+
+import (
+	"os"
+	"testing"
+
+	"gopkg.in/yaml.v3"
+)
+
+// TODO(cretz): To test:
+// * Env var actually sets CLI arg
+// * Get single and all for env
+// * Delete single and all for env
+// * List envs
```

Comment:
:+1: Would like to avoid as part of this PR due to size, but can definitely open issues (unsure if they are JIRA or GH) for filling out the tests on these TODOs once we merge here.

Reply:
That's fine; I have no opinion on whether Jira or GH is best. Jira's probably fine since it's more internal noise the community is unlikely to want to participate in, but there's no reason they can't be in GH if preferred.